code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -132,6 +132,39 @@
 	heal_threshold = 0
 	heal_amount = 5
 
+/mob/living/simple_animal/bot/medbot/nukie
+	name = "Oppenheimer"
+	desc = "A Nanotrasen medibot, stolen and upgraded by the Syndicate. Despite their best efforts at reprogramming, it still appears visibly upset near nuclear explosives."
```

Comment:
what makes normal medbots nanotrasen

Reply:
uh it's from a nt station or something. does this really mater

Reply:
> uh it's from a nt station or something. does this really mater

its not like this is a high priority pr or 24 hours passed so yeah why not

Reply:
```suggestion
	desc = "A medibot stolen from a Nanotrasen station and upgraded by the Syndicate. Despite their best efforts at reprogramming, it still appears visibly upset near nuclear explosives."
```